extensions/systemd/index.js

```diff
@@ -10,6 +10,26 @@ const {Extension, errors} = require('../../lib');
 const {ProcessError, SystemError} = errors;
 
 class SystemdExtension extends Extension {
+    migrations() {
+//        const migrations = require('./migrations');
+        return [{
+            before: '1.15.0',
+            title: 'Saving node binary',
+            skip: ({instance}) => instance.process.name !== 'systemd' || instance.nodeBinaryIsKnown(),
+            task: async ({instance}) => {
+                const serviceFilename = `ghost_${instance.name}.service`;
+                const fullFilePath = path.resolve(instance.dir, './system/files', serviceFilename);
+
+                const fileContents = await fs.readFile(fullFilePath, 'utf8');
+
+                const nodeBinaryLine = fileContents.split('\n').filter(line => line.startsWith('ExecStart'))[0];
```

Comment:
minor efficiency thing
```suggestion
                const nodeBinaryLine = fileContents.split('\n').find(line => line.startsWith('ExecStart'));
```